### DISCUSSION TITLE
Is there a config for getting a depNameLinked out of a customMatcher?

### DISCUSSION THREAD

**Discussion by childnode** [renovatebot/renovate#27269] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

`{{{depNameLinked}}}` typically used in prBody is not linked for any customMatcher as far as I see?

https://github.com/renovatebot/renovate/blob/83abc24ba86a7de2ceb69c12caf5f49e2d6f0db6/lib/workers/repository/update/pr/body/index.ts#L28

or is there anything how we can set `sourceUrl` etc? But I don't see a template for it https://docs.renovatebot.com/configuration-options/#custommanagers

needs an improvement or is it a bug because registry finding is not translated? In my case, it is a simple regexManager with `"datasourceTemplate": "artifactory"`, so I expected the datasource is relevant to extract the source information?

**Comment by viceice** [comment]
yes, datasource needs to return the source url
